common/src/test/java/io/netty/util/internal/NativeLibraryLoaderTest.java

```diff
@@ -51,6 +55,21 @@ public void testFileNotFoundWithNullClassLoader() {
         }
     }
 
+    @Test
+    void testMultipleResourcesInTheClassLoader() throws MalformedURLException {
+        URL url1 = new File("src/test/data/NativeLibraryLoader/1").toURI().toURL();
```

Comment:
I have intentionally used `src/test/data` here, because `src/test/resource` will contribute with the same resource via the parent class loader.

Reply:
Can you also add a test with different "resourceNames" to show it will pass ?

Reply:
Added `testSingleResourceInTheClassLoader`